### PR TITLE
add patient readmit mutation

### LINKED ISSUE #736: Patient Readmittance
### Describe the feature

re-admit discharged patients

### Why is this feature needed?

Currently, discharged patients cannot be readmitted just deleted

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #736

### CODE REVIEW

File: tasks/mutations/patient_mutations.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   GetPatientDetailsResponse,
   GetPatientAssignmentByWardRequest,
   GetPatientListRequest,
-  DeletePatientRequest
+  DeletePatientRequest, ReadmitPatientRequest
```
Comment (use-to) [inline tasks/mutations/patient_mutations.ts:15]:
```suggestion
  DeletePatientRequest,
  ReadmitPatientRequest
```

File: tasks/pages/ward/[id].tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ import type { PatientDTO, PatientMinimalDTO } from '../../mutations/patient_muta
 import {
   useAssignBedMutation,
   usePatientCreateMutation,
-  usePatientDischargeMutation,
+  usePatientDischargeMutation, useReadmitPatientMutation,
```
Comment (use-to) [inline tasks/pages/ward/[id].tsx:18]:
```suggestion
  usePatientDischargeMutation,
  useReadmitPatientMutation
```

**Review by use-to — APPROVED** [review]
Just some weird import formatting

**Review by use-to — CHANGES_REQUESTED** [review]
Dragging discharged patients onto empty bed cards is not working.

**Comment by DasProffi** [comment]
> Dragging discharged patients onto empty bed cards is not working.

Are you on the most recent version of the branch? For me, it works

**Comment by DasProffi** [comment]
> > Dragging discharged patients onto empty bed cards is not working.
> 
> Are you on the most recent version of the branch? For me, it works

Never mind, the error now also occurs for me, but seems timing related

**Review by use-to — APPROVED** [review]
Restarted Cloudflare build. LGTM now!

**Pull request merged**